app/views/admin/poll/questions/answers/_form.html.erb

```diff
@@ -1,15 +1,30 @@
-<%= form_for(@answer, url: form_url) do |f| %>
+<%= render "admin/shared/globalize_locales", resource: @answer %>
+
+<%= translatable_form_for(@answer, url: form_url) do |f| %>
 
   <%= render 'shared/errors', resource: @answer %>
 
   <%= f.hidden_field :question_id, value: @answer.question_id || @question.id %>
 
-  <%= f.text_field :title %>
+  <%= f.translatable_text_field :title %>
 
   <div class="ckeditor">
-    <%= f.cktext_area :description,
-                      maxlength: Poll::Question.description_max_length,
-                      ckeditor: { language: I18n.locale } %>
+    <%= f.label :description, t("admin.shared.description") %>
+    <% @answer.globalize_locales.each do |locale| %>
+      <% globalize(locale) do %>
+        <%= content_tag :span, class: "js-globalize-attribute",
+                               data: { locale: locale },
+                               style: display_translation?(locale) do %>
+
+          <%= f.cktext_area "description_#{locale}",
```

Comment:
Here we need to handle the case of non-underscored locale names, like `pt-BR`. For this you can use the [`#localized_attr_name_for(method, locale)` method](https://github.com/globalize/globalize-accessors/blob/master/lib/globalize-accessors.rb#L18) from the globalize-accessors gem (like [here](https://github.com/consul/consul/commit/c7fcdd9b0ec905063a803d93d45a454c826e14db#diff-4d169e27f1a8740fdcd328f352bf82d2R50)).

Reply:
I hope you like it https://github.com/consul/consul/pull/2914/commits/e6762d6bcacc9c9feb5d629b138f26bc59c5ac17